api/types/kustomization.go

```diff
@@ -116,7 +119,7 @@ type Kustomization struct {
 	// CRDs themselves are not modified.
 	Crds []string `json:"crds,omitempty" yaml:"crds,omitempty"`
 
-	// Deprecated.
+	// Deprecated: Replace with the Resources.
```

Comment:
Instead of adding a line, add "Deprecated:" to the existing message below.

Reply:
I fixed to below line.
https://github.com/kubernetes-sigs/kustomize/pull/4723/files#diff-fa94909235803d7e221a81bbf24d0005b59062f6f4b80110a1d2b390919379c7R122